website/docs/podman/gpu.md

```diff
@@ -43,12 +43,18 @@ Run the following commands **on the Podman Machine, not the host system**:
 
 ```sh
 $ curl -s -L https://nvidia.github.io/libnvidia-container/stable/rpm/nvidia-container-toolkit.repo | \
- sudo tee /etc/yum.repos.d/nvidia-container-toolkit.repo && \
- sudo yum install -y nvidia-container-toolkit && \
- sudo nvidia-ctk cdi generate --output=/etc/cdi/nvidia.yaml && \
+ tee /etc/yum.repos.d/nvidia-container-toolkit.repo && \
+ yum install -y nvidia-container-toolkit && \
+ nvidia-ctk cdi generate --output=/etc/cdi/nvidia.yaml && \
  nvidia-ctk cdi list
 ```
 
+:::info
+
+If you change the device or CUDA driver configuration, you must generate a new CDI specification. A configuration change can occur when MIG devices are created or removed, or when the driver is upgraded.
```

Comment:
```suggestion
A configuration change might occur when you create or remove Multi-Instance GPU (MIG) devices, or upgrade the Compute Unified Device Architecture (CUDA) driver. In such cases, you must generate a new Container Device Interface (CDI) specification. 
```

Reply:
The current info looks repetetive. Both the first and second sentences start from configuration change.